BUILD

```diff
@@ -0,0 +1,522 @@
+        "c++/src/kj/compat/url-test.c++",
+        "c++/src/kj/encoding-test.c++",
+        "c++/src/kj/filesystem-test.c++",
+        "c++/src/kj/function-test.c++",
+        "c++/src/kj/one-of-test.c++",
+        "c++/src/kj/parse/char-test.c++",
+        "c++/src/kj/parse/common-test.c++",
+        "c++/src/kj/refcount-test.c++",
+        "c++/src/kj/string-tree-test.c++",
+        "c++/src/kj/tuple-test.c++",
+        "c++/src/kj/units-test.c++",
+    ],
+    copts = [
+        "-Wno-shadow",
+        "-Wno-unused-parameter",
+        "-Wno-sign-compare",
+    ],
+    linkopts = ["-ldl"],
+    deps = [
+        ":kj",
+        ":kj-async",
+        ":kj-gzip",
+        ":kj-heavy-tests-hdrs",
+        ":kj-http",
+        ":kj-test",
+        ":kj-tls",
+    ],
+)
+
+cc_import(
+    name = "capnp_schemas",
+    hdrs = [
+        "c++/src/capnp/c++.capnp",
+        "c++/src/capnp/schema.capnp",
+        "c++/src/capnp/stream.capnp",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "capnp",
+    srcs = [
+        "c++/src/capnp/any.c++",
+        "c++/src/capnp/arena.c++",
+        "c++/src/capnp/blob.c++",
+        "c++/src/capnp/c++.capnp.c++",
+        "c++/src/capnp/dynamic.c++",
+        "c++/src/capnp/layout.c++",
+        "c++/src/capnp/list.c++",
+        "c++/src/capnp/message.c++",
+        "c++/src/capnp/schema.c++",
+        "c++/src/capnp/schema.capnp.c++",
+        "c++/src/capnp/schema-loader.c++",
+        "c++/src/capnp/serialize.c++",
+        "c++/src/capnp/serialize-packed.c++",
+        "c++/src/capnp/stream.capnp.c++",
+        "c++/src/capnp/stringify.c++",
+    ],
+    hdrs = [
+        "c++/src/capnp/any.h",
+        "c++/src/capnp/arena.h",
+        "c++/src/capnp/blob.h",
+        "c++/src/capnp/c++.capnp.h",
+        "c++/src/capnp/capability.h",
+        "c++/src/capnp/common.h",
+        "c++/src/capnp/compat/std-iterator.h",
+        "c++/src/capnp/dynamic.h",
+        "c++/src/capnp/endian.h",
+        "c++/src/capnp/generated-header-support.h",
+        "c++/src/capnp/layout.h",
+        "c++/src/capnp/list.h",
+        "c++/src/capnp/membrane.h",
+        "c++/src/capnp/message.h",
+        "c++/src/capnp/orphan.h",
+        "c++/src/capnp/pointer-helpers.h",
+        "c++/src/capnp/pretty-print.h",
+        "c++/src/capnp/raw-schema.h",
+        "c++/src/capnp/schema.capnp.h",
+        "c++/src/capnp/schema.h",
+        "c++/src/capnp/schema-lite.h",
+        "c++/src/capnp/schema-loader.h",
+        "c++/src/capnp/schema-parser.h",
+        "c++/src/capnp/serialize.h",
+        "c++/src/capnp/serialize-async.h",
+        "c++/src/capnp/serialize-packed.h",
+        "c++/src/capnp/serialize-text.h",
+        "c++/src/capnp/stream.capnp.h",
+    ],
+    copts = [
+        "-Wno-shadow",
+        "-Wno-unused-parameter",
+        "-iquote c++/src/kj",
+    ],
+    include_prefix = "capnp",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//:kj",
```

Comment:
All the deps for these targets should be revisited. This library does not directly depend on `//:kj`.